docs/src/main/sphinx/release/release-381.md

```diff
@@ -0,0 +1,102 @@
+# Release 381 (13 May 2022)
+
+## General
+
+* Add support for table functions. ({issue}`1839`, {issue}`11937`)
+* Add support for fault-tolerant execution with exchange spooling on Azure Blob Storage . ({issue}`12211`)
+* Increase the default number of stages allowed for a query from 100 to 150,
+  specified with `query.max-stage-count`, so large queries do not fail. ({issue}`12292`)
+* Use the `fault_tolerant_execution_partition_count` session property, or the
+  `fault-tolerant-execution-partition-count` configuration property, to adjust
+  the number of partitions for distributed joins and aggregations executed with
+  fault-tolerant execution enabled (`retry-policy=TASK`). ({issue}`12263`)
+* Rename configuration property used for selecting a policy for low memory query
+  killer from `query.low-memory-killer.policy` to
+  `query.low-memory-query-killer.policy`. ({issue}`12242`)
+* Introduce specific configuration property
+  `query.low-memory-task-killer.policy` for selecting a policy for low memory
+  task killer. The policy is used if task level retries are enabled
+  (`retry-policy=TASK`). ({issue}`12242`)
+* Introduce `least-wasted` low memory task killer policy. This policy avoids
+  killing tasks that are already executing for a long time, so a significant
```

Comment:
> "so a significant amount of work is not wasted"

It could still be a significant amount. This just minimizes how much work is wasted -- let's rephrase accordingly.

Reply:
Renamed to `least-waste`.